xilem/examples/calc.rs

```diff
@@ -3,16 +3,21 @@
 
 //! A simple calculator example
 #![expect(clippy::cast_possible_truncation, reason = "Deferred: Noisy")]
+#![expect(
+    clippy::shadow_unrelated,
+    reason = "Seems to prevent patterns we'd want to use."
+)]
```

Comment:
See #978, which would obviate this addition. I'd like that to land before this PR